utils/near-performance-metrics/src/stats_enabled.rs

```diff
@@ -126,11 +149,27 @@ impl ThreadStats {
                 thread_memory_count(tid),
                 self.c_mem,
             );
+            if self.write_buf_added.as_u64() > 0
+                || self.write_buf_capacity.as_u64() > 0
+                || self.write_buf_added.as_u64() > 0
+                || self.write_buf_drained.as_u64() > 0
+            {
+                info!(
+                    "        Write_buffer len: {} cap: {} added: {} drained: {}",
+                    self.write_buf_len,
+                    self.write_buf_capacity,
+                    self.write_buf_added,
+                    self.write_buf_drained,
+                );
+            }
+            self.write_buf_added = Default::default();
+            self.write_buf_drained = Default::default();
+
             let mut stat: Vec<_> = self.stat.iter().collect();
             stat.sort_by(|x, y| (*x).0.cmp(&(*y).0));
 
             for entry in stat {
-                warn!(
+                info!(
```

Comment:
why

Reply:
Those messages should have been `info` instead of `warn`. I would only use `warn`/`error`. For event's that aren't supposed to happen, like taking too much time, etc.

I moved this change out to https://github.com/near/nearcore/pull/4015